app/models/setting/auth.rb

```diff
@@ -4,7 +4,7 @@ def self.default_settings
       set('oauth_active', N_("Foreman will use OAuth for API authorization"), false, N_('OAuth active')),
       set('oauth_consumer_key', N_("OAuth consumer key"), '', N_('OAuth consumer key'), nil, {:encrypted => true}),
       set('oauth_consumer_secret', N_("OAuth consumer secret"), '', N_("OAuth consumer secret"), nil, {:encrypted => true}),
-      set('oauth_map_users', N_("Foreman will map users by username in request-header. If this is set to false, OAuth requests will have admin rights."), true, N_('OAuth map users')),
+      set('oauth_map_users', N_("Foreman will map users by username in request-header. If this is enabled, OAuth requests will have admin rights."), true, N_('OAuth map users')),
```

Comment:
Isn't it the other way?
```suggestion
      set('oauth_map_users', N_("When enabled, Foreman will map users by username in request-header. If this is disabled, OAuth requests will have admin rights."), true, N_('OAuth map users')),
```